openapi/kas-fleet-manager.yaml

```diff
@@ -1022,7 +1022,8 @@ components:
           type: boolean
           default: false
         supported_instance_types:
-          description: 'The Kafka instance types supported by this region.'
+          deprecated: true  
+          description: DEPRECATION NOTICE - instance_type will be deprecated
```

Comment:
Since the `deprecated` field is attached to this property, I don't think we need to change the description here. Let's keep this the same as before. 

Instead, we can update the description of the [endpoint](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/openapi/kas-fleet-manager.yaml#L429) if needed. Do you think this would be useful @wtrocki, @machi1990? Or is adding the deprecated field enough?

Reply:
Since we are not deprecating the endpoint but a field in the response returned by the endpoint, I'd go with the approach of modifying the field's specific description as done in the PR. 
My reasons for this are:
- The change on deprecation notice is localised. 
- That was when a field is removed in the future we won't have to think about updating the endpoint description

Reply:
I'm reluctant to completely replace the previous description in case someone looks at our api docs and wonder what these were used for since it's not completely removed just yet. 

Maybe we can have a multi-line description here with both the current description and the deprecation notice?

Reply:
There are no rules per se. If deprecated flag is set and field is simply going to be removed then there is no need to change description/do much else. The only exception is when we replacing one field with another. Then description might mention what is the new field to use

Reply:
> Maybe we can have a multi-line description here with both the current description and the deprecation notice?

+1